flytectl/cmd/create/execution_util.go

```diff
@@ -150,7 +153,12 @@ func recoverExecution(ctx context.Context, executionName string, project string,
 func createExecutionRequest(ID *core.Identifier, inputs *core.LiteralMap, envs *admin.Envs, securityContext *core.SecurityContext, authRole *admin.AuthRole, targetExecName string, targetExecutionCluster string) *admin.ExecutionCreateRequest {
 
 	if len(targetExecName) == 0 {
-		targetExecName = "f" + strings.ReplaceAll(uuid.New().String(), "-", "")[:19]
+		config := runtime.NewApplicationConfigurationProvider()
```

Comment:
Let's update flyteadmin first. Ideally, the ID should only be generated from the backend instead of the client.